pinniped-components/tanzu-auth-controller-manager/hack/generate-package-secret.test.sh

```diff
@@ -6,7 +6,6 @@
 set -euo pipefail
 
 MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-TF_ROOT="${MY_DIR}/../../.."
```

Comment:
There is also a `generate.sh` sibling that does:

```
# generate RBAC rules
"${TF_ROOT}/hack/tools/bin/controller-gen" paths=./controllers +rbac:roleName=tanzu-auth-controller-manager output:rbac:dir="${TANZU_AUTH_CONFIG_DIR}"
```
So this is still depending on `TF_ROOT`.

Reply:
Pointing out that it is also depositing files into `<TF_ROOT>/packages/tanzu-auth`

Reply:
Related to above comment https://github.com/vmware-tanzu/tanzu-framework/pull/3253/files#r969852402.